### PR TITLE
Added footer component

### PR DESCRIPTION
**Pull request by SkiF-ex**
Task: https://trello.com/c/FspmV2sW
View on base mockup:
Screenshot_5

### CODE REVIEW

**Comment by LordGemma** [comment]
Please create in the components folder a separate folder for footer component. In the future, you will have a mess of files without folders

**Comment by LordGemma** [comment]
Please always add your new component in a particular place where we can see the result of your work. Components without a place of realization will be rejected

**Comment by LordGemma** [comment]
image
If you add a logo to the footer it should be a link to the home page

File: src/components/footer/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './footer.css';
+
+export default function Footer () {
+    return (
+        <footer className="footer">
+            <h3 className="footer_logo">TheMovieDb</h3>
```
Comment (JIeCoRyI3) [inline src/components/footer/footer.jsx]: We need to import logo from somewhere, because we could want to reuse it

**Pull request merged**